targets/TARGET_NUVOTON/TARGET_M2351/flash_api.c

```diff
@@ -134,4 +134,12 @@ void flash_set_target_config(flash_t *obj)
 }
 
 #endif  // #if defined (__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U)
+
+MBED_NONSECURE_ENTRY uint8_t flash_get_erase_value(const flash_t *obj)
```

Comment:
flash_get_erase_value is added her as non-secure entry function. May I know why this API is different than rest of the API's in file?

It is causing build failures in generating secure side library :-(

Reply:
CC @ARMmbed/mbed-os-maintainers  @ARMmbed/team-nuvoton